hotel-management/src/components/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode } from 'react';
+import './index.css';
+
+type ErrorMessageProps = {
+  color: 'primary' | 'error';
```

Comment:
Why we need different color error message, this one should be consistent through the app